### ISSUE TITLE
Minify multiple JS files without combining

### ISSUE THREAD

**Issue by alireza19330** [eriwen/gradle-js-plugin#158]
I want to minify multiple JS files in a directory without combining them together. for example I have these files in my directory:
first.js, second.js, third.js ....
and I want to have the minified version of them on the same directory and the same files. 
Sorry, I'm not too familiar with Gradle and groovy syntax.

Is there any way to do so? I want to do something like that:

```
apply plugin: 'com.eriwen.gradle.js'
task MinifyJsFiles(type: com.eriwen.gradle.js.tasks.MinifyJsTask) {
    source = DIR_TO_ALL_JS_FILES
    dest = DIR_TO_PERSIST_OR_OVERRIDE_JS_FILES
    closure.warningLevel = 'QUIET'
}
```

Right now I handled that by defining several Gradle task for each minifying!

**Comment by alireza19330** [comment]
I found a good way here:
https://stackoverflow.com/a/40230421/826008
I think it's a good idea to merge this piece of code to the MinifyJS plugin

**Comment by rkrisztian** [comment]
That is not a good way at all, considering that:

* Gradle 4 already deprecates `Task.execute`, and Gradle 5 even removes it. (And this plugin hasn't been updated since 2017.)
* You may end up having hundreds of tasks, this may look really hard to read in the logs. And if you create these tasks in configuration time (which one of the SO answers suggest), your build script may become too slow.

It would be better to be able to tell what I want to minify and how using a closure mapper, for example...

**Comment by ChrisAcrobat** [comment]
Have there been any progress on this? Would it also be possible to obfuscate them separately?
I have separate roots in my project. It is functional to merge the files, but it is not optimal since the file gets bigger again with code that is never going to be used.

**Comment by ChrisAcrobat** [comment]
I saw this from @eriwen: https://stackoverflow.com/a/34187137
Have this been implemented yet, or do I still need to create a local version? I will experiment with this later.

**Comment by ChrisAcrobat** [comment]
Yes. I think I needed to do some changes to make it work for my project, but yes. I think it was remove ".min" and put in a directory instead.
